src/main/java/frc/robot/commands/DisposeCommand.java

```diff
@@ -1,23 +1,40 @@
 package frc.robot.commands;
 
+import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.subsystems.Acquisition;
 
 public class DisposeCommand extends Command {
     private Acquisition acquisition = Acquisition.getInstance();
+    private double secondsToDispose = 0.0;
+    private Timer timer = new Timer();
 
     public DisposeCommand() {
         super.addRequirements(acquisition);
     }
 
+    public DisposeCommand(double secondsToDispose) {
+        this.secondsToDispose = secondsToDispose;
+
+    }
+
     @Override
     public void execute() {
+        timer.restart();
         acquisition.dispose();
     }
 
     @Override
     public boolean isFinished() {
-        return !(acquisition.getTopLaser() || acquisition.getBottomLaser() || acquisition.getAlgaeSwitch());
+        if (!(acquisition.getTopLaser() || acquisition.getBottomLaser())) {
```

Comment:
```suggestion
        if (!(acquisition.getTopLaser() && acquisition.getBottomLaser())) {
```
We want to make sure both lasers are clear, not just 1 or the other